iterx.go

```diff
@@ -56,20 +58,34 @@ func (iter *Iterx) Unsafe() *Iterx {
 	return iter
 }
 
-// Get scans first row into a destination and closes the iterator. If the
-// destination type is a struct pointer, then StructScan will be used.
-// If the destination is some other type, then the row must only have one column
-// which can scan into that type.
+// Struct forces the iterator to treat a single-argument struct as non-scannable.
+// This is is useful if you need to scan a row into a struct that also implements gocql.Unmarshaler
+// or gocql.UDTUnmarshaler.
+func (iter *Iterx) Struct() *Iterx {
```

Comment:
It's not very consistent, I think we should name it like `DisableUDT` both field and func.